guides/schematics.md

```diff
@@ -30,9 +21,13 @@ This schematic will:
 - Adds Roboto fonts to your index.html
 - Apply simple CSS reset to body
 
-```
-ng add @angular/material
-```
+
+## Generator Schematics
```

Comment:
I don't know if "Generator Schematics" is the best term, but all the schematics are "packaged" so it seems like suboptimal name. I'd rather differentiate them by how they are used "install vs generator" than "shell vs packaged". please let me know if you think otherwise.